CHANGELOG.md

```diff
@@ -14,14 +14,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Breaking changes:
```

Comment:
While we're here we should probably update the date just above :D It's currently `2020-12-14`

Reply:
Good spot! I've submitted a second request to fix this